Dev/dhruvpra/readme init 23.07

dhruvpt opened the pull request.

dhruvpt commented on pkg/inframanager/p4/service.go:
```diff
@@ -633,38 +645,54 @@ func DeleteServiceRules(ctx context.Context, p4RtC *client.Client,
+		log.Debugf("Deleting from table AsSl3TcpTable, member ids: %v, mod blob ptrs: %v, group id: %d",
+			memberID, modblobPtrDNAT, groupID)
 		if err = AsSl3TcpTable(ctx, p4RtC, memberID, modblobPtrDNAT, groupID, Delete); err != nil {
 			return nil
```
Shouldn't this be return err instead of return nil. 
Same comment for all below calls.

dhruvpt commented on pkg/inframanager/p4/service.go:
```diff
@@ -633,38 +645,54 @@ func DeleteServiceRules(ctx context.Context, p4RtC *client.Client,
 		}
 	default:
 		log.Errorf("Invalid protocol type")
 		return nil
```
this too should return some error

venkateshpnv replied:
I think I made a typo while copying the lines. Will fix them.

venkateshpnv committed:
Fixed infraagent failure in insecure mode (#52)

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>
Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

Commit:
Dev/dhruvpra/k8s merge ipu target (#55)

* Merged files to add support for IPU target

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

* Remove IPU P4 artifacts, update Makefile for DPDK target and rename valueToBytes16 as ValueToBytes16 to make it global scope

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

* Reset the github actions to same as before for DPDK target.
Created target specific run-test.sh and infraagent-configmap.yaml files and copied the right one at build time.
Updated the inframanager-daemonset.yaml to remove log volume mounts.
Removed the iproute2-rdma from agent Docketfile.
Added mev tag to p4mev_suite_test.go file

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

* Global renaming and replacement of mev with es2k. Reverted removal of iproute2-rdma from agent docket image

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

* Revert back to ubuntu-latest for github actions

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

---------

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>
Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>
Co-authored-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>
Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

venkateshpnv committed:
Inframanager fixes for code merge. (#56)

* Infrmanager fixes for code merge.

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

* Set default interface to cdq and mtu size to 1500

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

* Retain old default arp-mac

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

---------

Signed-off-by: Neelakanta Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>
Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

Commit:
Upgrade to Go 1.20.5 (#57)

* Upgrade to Go 1.20.5

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

* Run git workflows for DPDK

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

---------

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>

Commit:
Inital README and Release Notes draft for IPDK 23.07 release

Signed-off-by: Tripathi, Dhruv Prakash <dhruv.tripathi@intel.com>
